staking_deposit/utils/ssz.py

```diff
@@ -132,3 +144,24 @@ class SignedBLSToExecutionChange(Serializable):
         ('message', BLSToExecutionChange),
         ('signature', bytes96),
     ]
+
+
+class VoluntaryExit(Serializable):
+    """
+    Ref: https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#voluntaryexit
+    """
+    fields = [
+        # TODO: Figure out correct type for epoch
```

Comment:
Remove. Epoch is uint64: https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#compute_domain

Reply:
ack